src/ameba/config.cr

```diff
@@ -24,7 +24,12 @@ class Ameba::Config
     json:     Formatter::JSONFormatter,
   }
 
-  PATH = ".ameba.yml"
+  DEFAULT_PATHS = {
+    "~/.ameba.yml",
+    "~/.config/ameba/config.yml",
```

Comment:
i think this must follow [XDG spec](https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html) and use the `${XDG_CONFIG_HOME:-~/.config}/ameba/` folder.

Reply:
something like this?

```cr
XDG_CONFIG_HOME = ENV.fetch("XDG_CONFIG_HOME", "~/.config")

DEFAULT_PATHS = {
  "~/.ameba.yml",
  File.expand_path("#{XDG_CONFIG_HOME}/ameba", home: true)
}
```

Reply:
Yeah, no need for `.expand_path` though, since it's being already done l8r on

Reply:
Fixed, thanks for spotting! 🙏